sites/docs/src/content/docs/tutorials/external_usage/nf-core_configs_outside_nf-core.md

```diff
@@ -7,84 +7,126 @@ When you've been working with nf-core pipelines for a while, you often will get
+
+   At the bare minimum, the file should contain:
+
+   ```groovy
+   process {
+       cpus   = 1
+       memory = 7.GB
+       time   = 4.h
+   }
+   ```
+
+   (or other sensible default values)
+
+   :::note{collapse title="Note on older nf-core template/Nextflow versions"}
+
+   If you wish to use pipelines generated with the pre-`v3.0.0` nf-core template, and/or when running with Nextflow versions earlier than 24.04.0 you may need to use an older syntax for setting resources. The following prevents the resources from exceeding a maximum limit.
+
+   Set in `conf/base.config` default values.
+
+   ```groovy
+   process {
+       cpus   = { check_max( 1    * task.attempt, 'cpus'   ) }
+       memory = { check_max( 7.GB * task.attempt, 'memory' ) }
+       time   = { check_max( 4.h  * task.attempt, 'time'   ) }
+   }
+   ```
+
+   Then add the [`check_max`](https://github.com/nf-core/tools/blob/0912990a63ef29e44e07cc2ba6ab81113684e0ae/nf_core/pipeline-template/nextflow.config#L233-L264) function to `nextflow.config`
+
+   The following should be placed outside all scopes, typically at the bottom of the file:
+
+   ```groovy
+   // Function to ensure that resource requirements don't go beyond
+   // a maximum limit
+   def check_max(obj, type) {
+     if (type == 'memory') {
+         try {
+             if (obj.compareTo(params.max_memory as nextflow.util.MemoryUnit) == 1)
+                 return params.max_memory as nextflow.util.MemoryUnit
+             else
+                 return obj
+         } catch (all) {
+             println "   ### ERROR ###   Max memory '${params.max_memory}' is not valid! Using default value: $obj"
+             return obj
+         }
+     } else if (type == 'time') {
+         try {
+             if (obj.compareTo(params.max_time as nextflow.util.Duration) == 1)
+                 return params.max_time as nextflow.util.Duration
+             else
+                 return obj
+         } catch (all) {
+             println "   ### ERROR ###   Max time '${params.max_time}' is not valid! Using default value: $obj"
+             return obj
+         }
+     } else if (type == 'cpus') {
+         try {
+             return Math.min( obj, params.max_cpus as int )
+         } catch (all) {
+             println "   ### ERROR ###   Max cpus '${params.max_cpus}' is not valid! Using default value: $obj"
+             return obj
+         }
+     }
+   }
+   ```
+
+   :::
+
+   For a more sophisticated `base.config`, see the full [nf-core template](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/conf/base.config)
+
+2. In a top level `nextflow.config`, specify two `param` to specify the URL where to look for nf-core/configs from.
+
+   ```groovy
+   custom_config_version      = 'master'
+   custom_config_base         = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
```

Comment:
I would write the `params` scope or write it as `params.custom_config_version` to make it clear
```suggestion
   params {
     custom_config_version      = 'master'
     custom_config_base         = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
   }
```